### ISSUE TITLE
SMA thread length insufficient

### ISSUE THREAD

**Issue by jordens** [sinara-hw/Kasli#74]
Still only 4 mm thread remaining  with washer, nut, isolation sleeve and panel. Needs to be 5.5 mm AFAICT.
Needs fixing. Options have been suggested and discussed.

**Comment by marmeladapk** [comment]
Isolators from GK ( https://github.com/sinara-hw/Kasli/issues/69 ) were used. You should be able to use SMA without the nut (and it should be stable).

**Comment by jordens** [comment]
Yes. But that wasn't done. It's either missing information in the BOM or at the assembly line.

**Comment by akaminska** [comment]
Both, actually. I already took care of the second and @marmeladapk can take care of the first for the next production run.
